mav_state_machine_msgs/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(mav_state_machine_msgs)
+
+find_package(catkin REQUIRED COMPONENTS message_generation std_msgs)
+include_directories(include ${catkin_INCLUDE_DIRS})
+
+
+add_message_files(
+  FILES
+  StartStopTask.msg
```

Comment:
You don't need to explicitly name these. add_message_files will automatically detect all messages in [/msg folder](https://github.com/ros/genmsg/blob/f9dad2f897fc9bafec038ed8a74986ba766e9e9d/cmake/genmsg-extras.cmake.em#L111). Makes it easier to maintain.